lms/djangoapps/certificates/tests/test_signals.py

```diff
@@ -21,11 +22,19 @@ def setUp(self):
         # Enable the feature
         CertificateGenerationConfiguration.objects.create(enabled=True)
 
-    def test_cert_generation_enabled_for_self_paced(self):
-        """ Verify the signal enable the self-generated certificates by default for
-        self-paced courses.
+
+    def test_cert_generation_flag_on_pacing_toggle(self):
+        """
+        Verify that signal enables or disables self-generated certificates
+        according to course-pacing.
         """
         self.assertFalse(certs_api.cert_generation_enabled(self.course.id))
```

Comment:
Please add comments before each step in this test, this would add to scenario readability.